string-manipulation-1/src/capitalize.ts

```diff
@@ -0,0 +1,12 @@
+/* exported capitalize */
+function capitalize(string: string): string {
+  const stringArr = string.split('');
+  for (let i = 0; i < stringArr.length; i++) {
+    if (i === 0) {
+      stringArr[i] = stringArr[i].toUpperCase();
+    } else {
+      stringArr[i] = stringArr[i].toLowerCase();
+    }
+  }
+  return stringArr.join('');
```

Comment:
Great job here! As an alternative solution we can actually do this with string methods rather than converting the string to and array and then back to a string. Here's how it would look:

```
function capitalize(word: string): string {
  return word[0].toUpperCase() + word.slice(1).toLowerCase();
}
```

Reply:
That is much more concise thank you